docs/resources/cluster.md

```diff
@@ -82,6 +83,7 @@ Property | Description | For Bundles | Default
 `kibana_node_size`|Desired master node size. See [here](https://developer.instaclustr.com/#operation/extendedProvisionRequestHandler) for more details. |Elasticsearch|
 `opensearch_dashboards_node_size`|Desired OpenSearch Dashboards node size. See [here](https://developer.instaclustr.com/#operation/extendedProvisionRequestHandler) for more details. |OpenSearch|
 `security_plugin`|Accepts true/false. Enables Security Plugin. This option gives an extra layer of security to the cluster. This will automatically enable internode encryption. Enforced to be true for all OpenSearch clusters. |Elasticsearch, OpenSearch|false
+`index_management_plugin`|Accepts true/false. Enables Index Management Plugin which allows automating index management activities. |Elasticsearch, OpenSearch|false
```

Comment:
Doesn't look like this is added to the schema?

Reply:
yes, I just added `index_management_plugin` and also `security_plugin` to schema